src/Microsoft.Azure.WebJobs.Host/Retry/RetryPolicyOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Represents retry options. 
+    /// </summary>
+    public sealed class RetryPolicyOptions : RetryResult
+    {
+        public RetryPolicyOptions() : base(0, TimeSpan.MinValue, false)
+        {
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether the retry options are defined.
+        /// </summary>
+        public bool IsDefined
+        {
+            get
+            {
+                return RetryCount != 0;
```

Comment:
Does this map to what is defined in the context? If so, isn't -1 the value that indicates it isn't defined?

Reply:
Actually we do not need the `IsDefined` property. I removed it.